### ISSUE TITLE
Nicer error message for conda index

### ISSUE THREAD

**Issue by bkreider** [conda/conda-build#348]
This should be a less opinionated message:

```
/opt/anaconda/bin/conda index
updating index in: /home/anaconda/pkgs/linux-32
Error:
    Indexing a copy of the Anaconda conda package channel is neither
    necessary nor supported.  If you which to add your own packages,
    you can do so by adding them to a separate channel.
```

**Comment by asmeurer** [comment]
Suggestions welcome.

**Comment by dan-blanchard** [comment]
Also, "which" should be "wish".  I didn't notice that until the third time I read that.

How about simply:

> Indexing a copy of the Anaconda conda package channel is not supported.  If you wish to add your own packages, you can do so by adding them to a separate channel.

If there's a supported way to make an Anaconda mirror, that might also be good to include in the message, since that's the only reason I could imagine that someone would reasonably want to do that.

**Comment by bkreider** [comment]
I got bitten by this again at a customer site.  They had this problem: https://github.com/conda/conda/issues/1498

When re-indexing we got this new error.  Luckily i had the conda-build source code in front of me and had them remove the _license* package.

**Comment by asmeurer** [comment]
This should work better now. conda index automatically converts the old metadata format to the new format.

**Comment by asmeurer** [comment]
Do you know what version of conda-build you were using? If it's a recent version, more information on the errors would be helpful.

**Comment by bkreider** [comment]
We installed miniconda and updated conda and conda-build to the latest.

**Comment by dhirschfeld** [comment]
Came across this because I accidentally copied the `_license*` bz2 to our internal mirror.

I don't understand what this error message is trying to say.  Mirroring the continuum repo *is* necessary for offline installs and it *is* supported by just copying the bz2 files and running a conda index.

I think a better message would be:
> *conda cannot index the _license\* file for whatever reason. Please remove this file and re-index*

**Comment by msarahan** [comment]
fixed by #2005
